docs/source/notebooks/tabular_tutorial.ipynb

```diff
@@ -0,0 +1,392 @@
+   "metadata": {},
+   "source": [
+    "Let's review some of the most likely label errors:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_raw.iloc[ranked_label_issues].assign(label=y_raw.iloc[ranked_label_issues]).head()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These datapoints appear the most suspicious to our model and should be carefully re-examined. Perhaps the original annotators missed something when deciding on the labels for these individuals.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **5. Train a more robust model from noisy labels**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For proper ML practice, let's split our data into train and test sets.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.model_selection import train_test_split\n",
+    "\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X_encoded, y, test_size=0.25, random_state=123)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We again standardize the numeric features, this time fitting the scaling parameters solely on the training set.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scaler = StandardScaler()\n",
+    "X_train[num_features] = scaler.fit_transform(X_train[num_features])\n",
+    "X_test[num_features] = scaler.transform(X_test[num_features])\n",
+    "\n",
+    "X_train = X_train.to_numpy()\n",
+    "y_train = y_train.to_numpy()\n",
+    "X_test = X_test.to_numpy()\n",
+    "y_test = y_test.to_numpy()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's now train and evaluate the original logistic regression model.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.metrics import accuracy_score\n",
+    "\n",
+    "clf.fit(X_train, y_train)\n",
+    "acc = clf.score(X_test, y_test)\n",
+    "print(f\"Test acuracy of original logistic regression: {acc}\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`cleanlab` provides a wrapper class that can be easily applied to any scikit-learn compatible model. Once wrapped, the resulting model can still be used in the exact same manner, but it will now train more robustly if the data have noisy labels.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.classification import LearningWithNoisyLabels\n",
+    "\n",
+    "clf = LogisticRegression()\n",
```

Comment:
In my newest version of tutorial I had replaced this line with:
`clf = LogisticRegression()  # Note we first re-initialize clf`

Can you update it to that? Thanks!

Reply:
done!